map_merge/src/map_merge.cpp

```diff
@@ -378,6 +389,41 @@ void MapMerge::executeposeEstimation()
   }
 }
 
+void MapMerge::publishTF()
+{
+  if (!tf_current_flag_.test_and_set()) {
+	// need to recalculate stored transforms
+	auto transforms = pipeline_.getTransforms();  // this is thread-safe access
+	//auto maps = getMaps();
+	tf_transforms_ = StampTransforms(transforms, world_frame_);
+  }
+  if (tf_transforms_.empty()) {
+	return;
+  }
+
+  // no locking. tf_transforms_ is always accessed only from tf thread
+  for (auto& transform : tf_transforms_) {
+	  tf_publisher_.sendTransform(transform);
+  }
+}
+
+std::vector<geometry_msgs::TransformStamped> MapMerge::StampTransforms(const std::vector<geometry_msgs::Transform> transforms_, const std::string& frame)
```

Comment:
functions start with lowercase letter. this function should be static inline non-member or static member as it does not access any member variables. names with trailing underscore indicate member variables and should not be used as arguments.

Reply:
This function actually uses two member variables, which allow proper map naming. I changed the name of the transforms parameter not to confuse. 
So should I leave this function as a member function as it uses two member variables or should I keep it static and pass the variables as parameters?